robotidy/transformers/RenameTestCases.py

```diff
@@ -36,11 +38,36 @@ class RenameTestCases(ModelTransformer):
         No Operation
     ```
 
+    ```
```

Comment:
We have two documentations: the one there that is 'short' doc displayed when someone runs ``robotidy --desc TRANSFORMER`` and the one available at readthedocs page. Here is the source file for the latter:
https://github.com/MarketSquare/robotframework-tidy/blob/main/docs/source/transformers/RenameTestCases.rst

Can you also update it? Just update first paragraph with this sentence: "If
    capitalize_each_word is true, will capitalize each word in test case name." and then add in the end of the file:

```
Capitalize each word
------------------------
If you set ``capitalize_each_word`` to ``True`` it will capitalize each word in the test case name::

     robotidy --transform RenameTestCases -c RenameTestCases:capitalize_each_word=True

.. tabs::

    .. code-tab:: robotframework Before

        *** Test Cases ***
        compare XML with json
            No Operation

    .. code-tab:: robotframework After

        *** Test Cases ***
        Compare XML With Json
            No Operation

```

If you want to see documentation live, install nox module and run following in the root of the repository:
```
nox -s docs
```
Documentation will generate in docs/_build/index.html